centos7-lcg96/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG REGISTRY=gitlab-registry.cern.ch/acts/machines
+ARG TAG=latest
+FROM ${REGISTRY}/centos7:${TAG}
+
+LABEL description="CERN CentOS 7 with Acts dependencies from LCG 96"
+LABEL maintainer="Paul Gessinger <paul.gessinger@cern.ch>"
+# increase whenever any of the RUN commands change
+LABEL version="2"
+
+ENV LCG_RELEASE 96
+ENV LCG_PLATFORM x86_64-centos7-gcc8-opt
+
+# Add the LCG rpm repository as described here
+#
+#     https://lcgpackages.web.cern.ch
+#
+# and use it to install all explicit Acts dependencies from an LCG release.
+RUN yum-config-manager --add-repo https://lcgpackages.web.cern.ch/lcgpackages/rpms/etc/yum.repos.d/lcg.repo \
+  && yum -y install \
+    LCG_${LCG_RELEASE}_Boost_1.70.0_${LCG_PLATFORM//-/_}.noarch \
+    LCG_${LCG_RELEASE}_CMake_3.14.3_${LCG_PLATFORM//-/_}.noarch \
+    LCG_${LCG_RELEASE}_cmaketools_1.8_${LCG_PLATFORM//-/_}.noarch \
+    LCG_${LCG_RELEASE}_DD4hep_01_10_${LCG_PLATFORM//-/_}.noarch \
+    LCG_${LCG_RELEASE}_doxygen_1.8.15_${LCG_PLATFORM//-/_}.noarch \
```

Comment:
I would remove this here, try and keep the images small.

Reply:
Doxygen has been removed.